### ISSUE TITLE
Interactive sky map based on our SDSS results

### ISSUE THREAD

**Issue by jeff-regier** [jeff-regier/Celeste.jl#288]
Something like [legacysurvey.org](http://legacysurvey.org/viewer)---perhaps a layer in legacysurvey.org itself.

**Comment by rcthomas** [comment]
Meeting with Kaylan Wednesday is set, what's the path to the TaskFarmered SDSS catalogs?

**Comment by jeff-regier** [comment]
The catalog is stored in
```
/global/cscratch1/sd/jregier/celeste_sdds_catalog_jld
```
That directory contains ~180,000 files in the `jld` format. Please see PR #496 for how to convert them to `fits` files.

**Comment by rcthomas** [comment]
@jeff-regier can you confirm these should indeed be zero-sized?  Otherwise I think I got the conversion done.

```
celeste-329.0000-329.2500-26.0000-26.2500.jld
celeste-322.7500-323.0000-49.7500-50.0000.jld
celeste-310.8750-311.0000-0.7500-0.8750.jld
celeste-307.7500-308.0000-8.2500-8.5000.jld
celeste-212.0000-212.5000-44.5000-45.0000.jld
celeste-197.0000-197.5000-43.5000-44.0000.jld
celeste-101.7500-102.0000-10.7500-11.0000.jld
```

**Comment by jeff-regier** [comment]
Yes, that's fine if a few are zero sized. Glad to hear the conversion went smoothly.

**Comment by jeff-regier** [comment]
@rcthomas Is this still something you're working on? It would help us to publicize the GB results. We'll also have "model" images once #663 is done.

**Comment by rcthomas** [comment]
Hi @jeff-regier no, I won't have time for this anymore sorry.  Kaylan's got all the info that's needed to ingest catalogs, but you will need to think about how to do the image modeling, I never looked at the viewer code.
